karma.conf.js

```diff
@@ -1,25 +1,31 @@
-const webpackConfig = require('./webpack.config');
+const commonjs = require('rollup-plugin-commonjs');
+const includePaths = require('rollup-plugin-includepaths');
+const rollupConfig = require('./rollup.config.js');
+
+rollupConfig.plugins.push(
+  commonjs(),
+  includePaths({})
+);
+
+rollupConfig.format = 'umd';
 
 module.exports = function(config) {
   config.set({
     files: [
-      'tests.webpack.js'
+      'src/__tests__/*.js',
+      'src/*.js',
     ],
-    
+
     preprocessors: {
-      'tests.webpack.js': [ 'webpack', 'sourcemap' ]
+      'src/*.js': ['rollup'],
+      'src/__tests__/*.js': ['rollup'],
     },
-    
+    rollupPreprocessor: rollupConfig,
+
     reporters: ['mocha'],
-    
+
     frameworks: ['mocha', 'chai', 'sinon'],
-    
+
     browsers: ['Chrome'],
```

Comment:
Rewriting from webpack to rollup. I have trouble starting the tests on my machine. Maybe another discussion on where to take the test setup.

Reply:
FWIW, we've been switching some of our other repos to Jest for tests.

Reply:
Yeah. I checked the tests here. Feels like the biggest hurdle are the http mocks. Could be worked around by refactoring to fetch if possible.

Reply:
Oh right, we're still using `XMLHttpRequest` here aren't we. We still need to support not-so-modern browsers here. Maybe we could switch to a fetch polyfill though.